subprojects/docs/src/samples/userguide/wrapper/customized-task/wrapperCustomized.sample.conf

```diff
@@ -1,2 +1,10 @@
-executable: gradle
-args: tasks
+commands: [{
+    execution-subdirectory: groovy
```

Comment:
Could you pls explain this configuration to me?
I assume that these are `GradleRunner` parameters, right?
But where will these be used?

Reply:
We have a tool called Exemplar (to be open-sourced soon) that is configured to execute these samples through configuration in these files.